code/modules/unit_tests/combat_stamina.dm

```diff
@@ -0,0 +1,20 @@
+/// Tests 100 stamina damage = stamcrit
+/datum/unit_test/stamcrit
```

Comment:
any tests that intentionally sleep ought to be put into the slow tests categories